Where2Play/Models/Setlist.cs

```diff
@@ -6,214 +6,213 @@
+    public Artist With { get; set; }
 
-    public partial class Venue
-    {
-        [JsonProperty("id")]
-        public string Id { get; set; }
+    [JsonProperty("tape", NullValueHandling = NullValueHandling.Ignore)]
+    public bool? Tape { get; set; }
+}
 
-        [JsonProperty("name")]
-        public string Name { get; set; }
+public partial class Venue
+{
+    [JsonProperty("id")]
+    public string Id { get; set; }
 
-        [JsonProperty("city")]
-        public City City { get; set; }
+    [JsonProperty("name")]
+    public string Name { get; set; }
 
-        [JsonProperty("url")]
-        public Uri Url { get; set; }
-    }
+    [JsonProperty("city")]
+    public City City { get; set; }
 
-    public partial class City
-    {
-        [JsonProperty("id")]
-        [JsonConverter(typeof(ParseStringConverter))]
-        public long Id { get; set; }
+    [JsonProperty("url")]
+    public Uri Url { get; set; }
+}
 
-        [JsonProperty("name")]
-        public string Name { get; set; }
+public partial class City
+{
+    [JsonProperty("id")]
+    [JsonConverter(typeof(ParseStringConverter))]
+    public long Id { get; set; }
 
-        [JsonProperty("state")]
-        public string State { get; set; }
+    [JsonProperty("name")]
+    public string Name { get; set; }
 
-        [JsonProperty("stateCode")]
-        public string StateCode { get; set; }
+    [JsonProperty("state")]
+    public string State { get; set; }
 
-        [JsonProperty("coords")]
-        public Coords Coords { get; set; }
+    [JsonProperty("stateCode")]
+    public string StateCode { get; set; }
 
-        [JsonProperty("country")]
-        public Country Country { get; set; }
-    }
+    [JsonProperty("coords")]
+    public Coords Coords { get; set; }
 
-    public partial class Coords
-    {
-        [JsonProperty("lat")]
-        public double Lat { get; set; }
+    [JsonProperty("country")]
+    public Country Country { get; set; }
+}
 
-        [JsonProperty("long")]
-        public double Long { get; set; }
-    }
+public partial class Coords
+{
+    [JsonProperty("lat")]
+    public double Lat { get; set; }
 
-    public partial class Country
-    {
-        [JsonProperty("code")]
-        public string Code { get; set; }
+    [JsonProperty("long")]
+    public double Long { get; set; }
+}
 
-        [JsonProperty("name")]
-        public string Name { get; set; }
-    }
+public partial class Country
+{
+    [JsonProperty("code")]
+    public string Code { get; set; }
 
-    public partial class Welcome
-    {
-        public static Welcome FromJson(string json) => JsonConvert.DeserializeObject<Welcome>(json, Where2Play.Models.Converter.Settings);
-    }
+    [JsonProperty("name")]
+    public string Name { get; set; }
+}
 
-    public static class Serialize
-    {
-        public static string ToJson(this Welcome self) => JsonConvert.SerializeObject(self, Where2Play.Models.Converter.Settings);
-    }
+public partial class Welcome
```

Comment:
Ok, here's an opportunity for a code review improvement...
Class names should be accurately descriptive of what they do.  "Welcome" is the default name generated by QuickType, if you don't specify another name.  It is not descriptive of the class itself.  So, what is a better name for this class?